### PR TITLE
perf: reduce calls of agencyList api

### PR DESCRIPTION
**Pull request by shootermv**
# Description
solves multiple calls to getAgenciesList api #346 

## screenshots
image

### CODE REVIEW

File: src/model/operator.ts [hunk 1]
```diff
@@ -5,22 +5,27 @@ export type Operator = {
   id: string
 }
 
-async function getOperatorId(name: string) {
+export const RELEVANT_OPERATORS = async () => {
```
Comment (NoamGaash) [inline src/model/operator.ts:8]: you can use IIFE to make `RELEVANT_OPERATORS` a promise. it could be nice, because it's reusable this way.

**Review by NoamGaash — COMMENTED** [review]
could you please explain why it had two calls in the first place? just for my understanding

File: src/model/operator.ts (hunk 1 above)
Comment (shootermv) [reply src/model/operator.ts:8]: Ta first place it was 8 calls. Like screenshot at issue

File: src/pages/components/OperatorSelector.tsx [hunk 2]
```diff
@@ -18,7 +18,7 @@ const OperatorSelector = ({
   const [operators, setOperators] = useState<Operator[]>([])
   useEffect(() => {
     const majorOperatorsIds = ['3', '5', '15', '18', '25']
-    RELEVANT_OPERATORS.then((resultObj) =>
+    RELEVANT_OPERATORS().then((resultObj) =>
```
Comment (NoamGaash) [inline src/pages/components/OperatorSelector.tsx:21]: now, when it's a function and not a constant, using camalCase would be better
```suggestion
    getRelevantOperators().then((resultObj) =>
```

**Review by NoamGaash — APPROVED** [review]

File: src/model/operator.ts (hunk 1 above)
Comment (NoamGaash) [reply src/model/operator.ts:8]: I see that it had many calls, I can't see why

**Comment by shootermv** [comment]
> could you please explain why it had two calls in the first place? just for my understanding

the reason for 2 calls is - Dashboard page has two instances of `OperatorSelector` each one shown at different viewport (personally i dont see the reason for such behavior)

**Pull request merged**